Project.toml

```
@@ -39,9 +37,8 @@ julia = "1.6"
 
 [extras]
 PkgVersion = "eebad327-c553-4316-9ea0-9fa01ccd7688"
-SimpleWeightedGraphs = "47aef6b3-ad0c-573a-a1e2-d07658019622"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["PkgVersion", "SimpleWeightedGraphs", "SparseArrays", "Test"]
+test = ["PkgVersion", "SparseArrays", "Test"]
```

```suggestion
test = ["SparseArrays", "Test"]
```
PkgVersion is not used anymore.